build.gradle

```diff
@@ -23,5 +23,6 @@ subprojects {
     repositories {
         jcenter()
         google()
+        maven { url 'https://jitpack.io' }
```

Comment:
Was this intentional? For the compileTestingKotlin dep?

Reply:
Yes this is required for the Kotlin testing library.

Reply:
I'll update with a comment here

Reply:
Updated